sharding-core/sharding-core-execute/src/main/java/org/apache/shardingsphere/core/execute/sql/execute/result/QueryResultUtil.java

```diff
@@ -53,8 +53,6 @@ public static Object getValue(final ResultSet resultSet, final int columnIndex)
     public static Object getValueByColumnType(final ResultSet resultSet, final int columnIndex) throws SQLException {
         ResultSetMetaData metaData = resultSet.getMetaData();
         switch (metaData.getColumnType(columnIndex)) {
-            case Types.BIT:
```

Comment:
Why not just put `Types.BIT` behind `Types.BOOLEAN` ?

Reply:
You can find the answer from #2460 and #2467